apps/right-to-rent-check/acceptance/features/tenant-details-change-link.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+
+Feature('Tenant Add Another Page Details Change links');
+
+Before((
+  I,
+  tenantAddAnotherPage
+) => {
+  I.visitPage(tenantAddAnotherPage, steps);
```

Comment:
As before, you may need to add an actual tenant in this step as this may be just creating an empty tenant